CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 5.6.0
+
+* Fix a bug in Send a file by email. Add support for an optional `is_csv` parameter in the `prepare_upload()` function. This ensures that the file is downloaded as a CSV rather than a TXT file.
```

Comment:
Final edit. Just spoke with Karl and we think the sentences are too disjoint so we now suggest:

Add support for an optional is_csv parameter in the prepare_upload() function. This fixes a bug when sending a CSV file by email. This ensures that the file is downloaded as a CSV rather than a TXT file.

After that then this is definitely going to be good to go. Thanks for your patience!